examples/closure/.gitignore

```diff
@@ -0,0 +1,10 @@
+node_modules/.bin
+
+node_modules/plovr
+
+node_modules/todomvc-app-css/*
+!node_modules/todomvc-app-css/index.css
+
+node_modules/todomvc-common/*
+!node_modules/todomvc-common/base.css
+!node_modules/todomvc-common/base.js
```

Comment:
No big deal but maybe give this a final newline, as stated by our editorconfig: https://github.com/tastejs/todomvc/blob/master/.editorconfig#L8